submissions/sprint_a/app.py

```diff
@@ -0,0 +1,10 @@
+from . import app
```

Comment:
Missing module docstring describing purpose of file.
- [Description](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings)
- [Example](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)